marklogic-data-hub/src/main/java/com/marklogic/hub/impl/EntityManagerImpl.java

```diff
@@ -101,9 +101,8 @@ public boolean saveQueryOptions() {
                 logger.info("Wrote entity-specific search options for Explorer to: " + stagingFile.getAbsolutePath());
                 FileUtils.writeStringToFile(expFinalFile, expOptions, Charset.defaultCharset());
                 logger.info("Wrote entity-specific search options for Explorer to: " + finalFile.getAbsolutePath());
+                return true;
```

Comment:
With this change we return false if only "options" were generated. Is that ok?

Reply:
Good catch. I'll adjust this to checkout that neither are null.

Reply:
I adjusted the logic to check that neither options were null.